tests/infrastructure/test_ialirt_ingest_lambda_construct.py

```diff
@@ -30,6 +30,66 @@ def populate_algorithm_table(setup_dynamodb):
     return items
 
 
+@pytest.fixture
+def populate_data_table(setup_data_table):
+    """Populate DynamoDB table."""
+    data_table = setup_data_table["data_table"]
+    items = [
+        {
+            "instrument": "mag",
+            "met_in_utc": "2021-01-01T00:00:00.000000001",
```

Comment:
Another thing that will help here is if we can truncate the times to seconds to reduce storage space, I'm not sure where this would come in up above though. I don't think we have any resolutions more than every 4 seconds here.